File: Python/Day 4/is_palindrome.py
test_list = ["abcde", "racecar", "hello", "noon"]

new_word = ""
for word in test_list:
	#for letter in word:
	for count in range(len(word)-1, -1, -1):
		print(word[count])
		#word[count] += new_word 
	print(new_word)

File: JavaScript/Day 4/isPalindrome.js
testList = ["madam", "racecar", "hello", "noon"];

function IsPerfectSquareFunction(testList){
	for (let count = 0; count < testList.length; count++){
		let wordInList = testList[count]
		let newWord = ""
		for (let counter = wordInList.length-1; counter >= 0; counter--){
			newWord += wordInList[counter];
		}
		//console.log(newWord);
		testList[count] = wordInList == newWord;
	}
	return testList
}
console.log(IsPerfectSquareFunction(testList));